src/viewer/Heatmap.ts

```diff
@@ -5,349 +5,126 @@ declare var InteractiveDataDisplay: any;
-                                viewState["probesViewModel"].selectProbe(-1);
-                            }
-                            else {
-                                $(".checkButton").removeClass("checkButton-checked");
-                                showSimilarBtn.addClass("checkButton-checked");
-                                viewState["uncertaintyRange"] = { min: lb, max: ub, probeid: probe.id };
-                                viewState["probesViewModel"].selectProbe(probe.id);
-                            }
-                        });
-
-                        if (probe !== undefined && (viewState["uncertaintyRange"] !== undefined && probe.id === viewState["uncertaintyRange"].probeid)) {
-                            $(".checkButton").removeClass("checkButton-checked");
-                            showSimilarBtn.addClass("checkButton-checked");
-                            viewState["uncertaintyRange"] = { min: lb, max: ub, probeid: probe.id };
-                        }
-
-                        $($("<span style='margin-left:3px;'>highlight similar</span>")).appendTo(checkBoxCnt);
-
-                        return result;
-                    }
-                }
-                return undefined;
-            }
-
-            var callback = function (vs, propName, extraData) {
-                if (plots === undefined && plots[0] === undefined && plots[1] === undefined) {
-                    viewState.unsubscribe(this);
-                    return;
-                }
-
-                if (plots[1].f_median === undefined || plots[1].f_lb === undefined || plots[1].f_ub === undefined) {
-                    return;
-                }
-
-                if (propName === "uncertaintyRange") {
-                    var range = vs.uncertaintyRange;
-
-                    if (range === undefined) {
-                        plots[1].host.css("visibility", "hidden");
-                        return;
-                    }
-
-                    var fmedian = plots[1].f_median;
-                    var shadeData = new Array(fmedian.length);
-                    for (var i = 0; i < fmedian.length; i++) {
-                        var fmedian_i = fmedian[i]
-                        shadeData[i] = new Array(fmedian_i.length);
-                        for (var j = 0; j < fmedian_i.length; j++) {
-                            shadeData[i][j] = (plots[1].f_lb[i][j] < range.max && plots[1].f_ub[i][j] > range.min) ? 0 : 1;
-                        }
-                    }
-                    plots[1].draw({ x: plots[1].x, y: plots[1].y, f: shadeData });
-                    plots[1].host.css("visibility", "visible");
-                }
-            }
-            if (plots !== undefined && plots[0] !== undefined && plots[1] !== undefined && viewState !== undefined) viewState.subscribe(callback);
-
-            return plots;
+            var plots = [heatmap];
+            //var getHeader = function () {
```

Comment:
@ElenaPochernina  No need in comments; we aren't going to return to the code. Just delete, if you don't mind.